src/word_wrap/Wrapper.java

```diff
@@ -3,9 +3,9 @@
 public class Wrapper {
 
     public static String wrap(String s, int column) {
-        if (s.length() < column) {
+        if (s.length() <= column) {
             return s;
         }
-        return s.substring(0, column) + "\n" + s.substring(column, s.length());
+        return s.substring(0, column) + "\n" + wrap(s.substring(column, s.length()), column);
```

Comment:
No, recursion seems to be a better smaller step than iteration.
Based on Uncle Bob's Transformation Priority Premise (https://blog.8thlight.com/uncle-bob/2013/05/27/TheTransformationPriorityPremise.html), `statement->recursion` is given precedence over `if->while`